Library/Formula/python3.rb

```diff
@@ -306,15 +310,11 @@ def caveats
 end
 
 __END__
-# Homebrew's tcl-tk is build in a standard unix fashion (due to link errors)
-# and we have to stop python from searching for frameworks and link against
-# X11.
-
 diff --git a/setup.py b/setup.py
-index d4183d4..9f69520 100644
+index 2779658..902d0eb 100644
```

Comment:
Just in keeping with our vague "patches get submitted upstream, get `inreplace`d or get removed" policy I think it would be worthwhile.